ios/Sources/HomeFeature/HomeListFeature.swift

```diff
@@ -0,0 +1,70 @@
+import ComposableArchitecture
+import Model
+import Repository
+
+public struct HomeListState: Equatable {
+    public var feedContents: [FeedContent]
+    public var message: String {
+        "Finished! 🤖"
+    }
+
+    public var topic: FeedContent? {
+        feedContents.first
+    }
+
+    public var listFeedContents: [FeedContent] {
+        Array(feedContents.dropFirst())
+    }
+
+    public init(feedContents: [FeedContent] = []) {
+        self.feedContents = feedContents
+    }
+}
+
+public enum HomeListAction {
+    case selectFeedContent
+    case tapFavorite(FeedContent)
```

Comment:
I want to make sure that we would pass only the necessary value (which is id & isFavorited) or whole struct.
I think both looks fine and I wanna leave a decision up to you.
(If you decide to use whole struct, please fix the current tapFavorite closure arguments in other modules :bow:)